docs/hellonode.md

```diff
@@ -126,6 +126,25 @@ Create a cluster via the Console: *Compute > Container Engine > Container Cluste
 
 It’s now time to deploy your own containerized application to the Kubernetes cluster! Please ensure that you have [configured](https://cloud.google.com/container-engine/docs/before-you-begin#optional_set_gcloud_defaults) `kubectl` to use the cluster you just created.
 
+**The rest of this document requires both the kubernetes client and server version to be 1.2.**  This should be true by default for new Google Container Engine clusters.
+
+```shell
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.0", GitCommit:"5cb86ee022267586db386f62781338b0483733b3", GitTreeState:"clean"}
+Server Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.0", GitCommit:"5cb86ee022267586db386f62781338b0483733b3", GitTreeState:"clean"}
+```
+
+**If either the client or server version is less than 1.2, see the [1.1 version of this document](https://github.com/kubernetes/kubernetes.github.io/blob/release-1.1/docs/hellonode.md).**
```

Comment:
Yes, please include this explanation. This is a recent change and it's
likely someone will be on an older version. The user won't know that the
instructions have changed.
On Mar 29, 2016 6:28 PM, "johndmulhausen" <notifications@github.com> wrote:

> In docs/hellonode.md
> <https://github.com/kubernetes/kubernetes.github.io/pull/248#discussion_r57826518>
> :
>
> > @@ -126,6 +126,25 @@ Create a cluster via the Console: *Compute > Container Engine > Container Cluste
> >
> >  It’s now time to deploy your own containerized application to the Kubernetes cluster! Please ensure that you have [configured](https://cloud.google.com/container-engine/docs/before-you-begin#optional_set_gcloud_defaults) `kubectl` to use the cluster you just created.
> >
> > +**The rest of this document requires both the kubernetes client and server version to be 1.2.**  This should be true by default for new Google Container Engine clusters.
> > +
> > +```shell
> > +$ kubectl version
> > +Client Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.0", GitCommit:"5cb86ee022267586db386f62781338b0483733b3", GitTreeState:"clean"}
> > +Server Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.0", GitCommit:"5cb86ee022267586db386f62781338b0483733b3", GitTreeState:"clean"}
> > +```
> > +
> > +**If either the client or server version is less than 1.2, see the [1.1 version of this document](https://github.com/kubernetes/kubernetes.github.io/blob/release-1.1/docs/hellonode.md).**
>
> GKE will default them to 1.2, yes. I think it'll be pretty slick soon with
> a drop-down for different versions of the documentation. If we want to be
> crystal clear and just leave the first line with a link to 1.1/1.0, I'll
> grumble but accept that compromise. But I promise, it's something I'm
> trying to pin down and soon. FYI, I even have 1.1 version stood up here:
>
> http://kubernetes-v1-1.github.io/
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes.github.io/pull/248/files/856a4f423b17abc13bdcd7ce93439f178dfddd67#r57826518>
>